homeassistant/components/transmission/sensor.py

```diff
@@ -68,26 +63,14 @@ def should_poll(self):
         """Return the polling requirement for this sensor."""
         return False
 
-    @property
-    def unit_of_measurement(self):
-        """Return the unit of measurement of this entity, if any."""
-        return self._unit_of_measurement
-
     @property
     def available(self):
         """Could the device be accessed during the last update call."""
         return self._tm_client.api.available
 
-    @property
-    def device_state_attributes(self):
-        """Return the state attributes, if any."""
-        if self._tm_client.api.started_torrent_dict and self.type == "started_torrents":
-            return {STATE_ATTR_TORRENT_INFO: self._tm_client.api.started_torrent_dict}
-        return None
-
     async def async_added_to_hass(self):
         """Handle entity which will be added."""
-        self.unsub_update = async_dispatcher_connect(
+        self._unsub_update = async_dispatcher_connect(
```

Comment:
Please use `self.async_on_remove` instead to register the unsubscribe function to be called on entity removal.
```py
self.async_on_remove(async_dispatcher_connect(...))
```